examples/adminapi.py

```diff
@@ -152,7 +152,12 @@ def example_create_acls(a, args):
         )
     ]
 
-    fs = a.create_acls(acl_bindings, request_timeout=10)
+    fs = None
+    try:
+        fs = a.create_acls(acl_bindings, request_timeout=10)
+    except ValueError as e:
+        print(f"Invalid input: {e}")
```

Comment:
The error string itself should tell what the error is, which isn't necessarily "Invalid input", but could be other things (in the future). So recommend just printing the error, or something like: `f"create_acls() failed: {e}"`